ports/geckolib/glue.rs

```diff
@@ -2216,14 +2214,14 @@ fn fill_in_missing_keyframe_values(all_properties:  &[TransitionProperty],
         return;
     }
 
-    let keyframe = match offset {
-        0. => unsafe {
+    let keyframe = if is_initial {
```

Comment:
Actually, I had not noticed that this didn't land yet, so maybe we should just require from Manish to do that damn enum now.